android/src/fdroid/play/listings/hu-HU/release-notes.txt

```diff
@@ -1,8 +1,10 @@
-• Új OpenStreetMap térképadatok (március 29)
-• Kísérleti sávnavigációs funkció
-• A felhasználó helyzetét jelző nyíl pozícióját megváltoztattuk, a jobb élmény érdekében
-• Megjavítottuk a könyvjelzők színválasztóját a jobb oldalról balra haladó írásmódú nyelveknél
-• Immáron a sebesség és a távolság helyesen van megjelenítve navigáció közben
-• Sok új bolttípussal bővült alkalmazásunk
-• Hozzáadtuk a pénz átutalóhelyeket, bicikli töltőállomásokat, és erőműveket
-…további infó: omaps.app/news
+• Új OpenStreetMap adatok (május 3)
+• A keresés most váltakozó páratlan és páros címeket talál
```

Comment:
Please change this to "A keresés funkció már támogatja az addr:interpolation címeket" as this sentence in the current form doesn't make sense.